client/html/pages/panel/settings/index.html

```diff
@@ -25,7 +25,7 @@
 
                 <!-- BEGIN CONTACTS NAV -->
                 <div class="col-sm-5 col-md-4 col-lg-3">
-                  <ul class="nav nav-pills nav-stacked nav-tab" data-toggle="tabs">
+                  <ul class="nav nav-pills nav-stacked nav-tabs-vert" data-toggle="tabs">
```

Comment:
This is how it looks like if I use "nav-tabs" to render it:
![image](https://cloud.githubusercontent.com/assets/4422230/19129452/b71551a6-8b1d-11e6-8718-ca448a3b64ae.png)

And this is what happens if I use the same structure as used [here](https://github.com/caike08/BancoTalentosTCC/blob/updates/client/html/pages/panel/jobs/index.html#L25):
![image](https://cloud.githubusercontent.com/assets/4422230/19129655/74288c2c-8b1e-11e6-9fdc-09b03a1469d7.png)

The second one is almost there, but it adds up a border in the last child. So I'm using a different one just for when a panel is inside settings. Got it?